docs/how-to/setup-time-sources.md

```diff
@@ -0,0 +1,28 @@
+# How to: Setup Time Sources
+
+Chrony alone cannot provide time; it requires another time source, either a 
+reference clock or another NTP server, to supply time to Chrony itself. 
+Currently, the Chrony charm supports only using another NTP server as the 
+time source.
+
+## Use Another NTP Server
+
+Provide the IP address or domain name of the NTP server to be used as the time 
+source in URL format, for example, `ntp://ntp.example.com`. 
+Pass this URL to the Chrony charm using the `sources` charm configuration. 
+The domain in the URL will be treated as a pool address for the NTP server.
+
+You can adjust the time source using 
+[pool options listed in the Chrony configuration](https://chrony-project.org/doc/4.6.1/chrony.conf.html). 
+These options are passed to the Chrony charm as URL parameters. 
+For example, to enable `iburst` and set `maxsources` to 2, use the 
+URL `ntp://ntp.example.com?iburst=true&maxsources=2`. 
+The only exception option is NTS, as explained in the next section.
+
+## Use Another NTS Server with NTS Enabled
```

Comment:
I recommend updating this heading to `## Use another NTS server with NTS enabled`